descontinued_medication.go

```diff
@@ -0,0 +1,151 @@
+package elation
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type DiscontinuedMedicationServicer interface {
+	Create(ctx context.Context, create *DiscontinuedMedicationCreate) (*DiscontinuedMedication, *http.Response, error)
+	Find(ctx context.Context, opts *FindDiscontinuedMedicationsOptions) (*Response[[]*DiscontinuedMedication], *http.Response, error)
+	Get(ctx context.Context, id int64) (*DiscontinuedMedication, *http.Response, error)
+	Update(ctx context.Context, id int64, update *DiscontinuedMedicationUpdate) (*DiscontinuedMedication, *http.Response, error)
+}
+
+var _ DiscontinuedMedicationServicer = (*DiscontinuedMedicationService)(nil)
+
+type DiscontinuedMedicationService struct {
+	client *Client
+}
+
+type DiscontinuedMedicationCreate struct {
+	MedOrder             int64  `json:"med_order"`
+	DiscontinueDate      string `json:"discontinue_date,omitempty"`
+	Reason               string `json:"reason,omitempty"`
+	IsDocumented         bool   `json:"is_documented"`
```

Comment:
It looks like this one supports a null, should it also be `omitempty`?

Reply:
Good catch, will change.